src/scss/helpers/_mq.scss

```diff
@@ -30,11 +30,33 @@
   }
 }
 
+@mixin fixed-width() {
```

Comment:
Are these necessary? We don't support ie7 or 8 anymore. Is fixed-width actually necessary?

Reply:
No. I'd rather remove all this on it's own branch.